Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -699,3 +699,18 @@
         state: pda-detective
   - type: Icon
     state: pda-detective
+
+- type: entity
+  parent: BasePDA
+  id: CBURNPDA
+  name: CBURN PDA
+  description: Smells like rotten flesh.
+  components:
+  - type: PDA
+    id: CBURNIDcard
+  - type: Appearance
+    visuals:
+    - type: PDAVisualizer
+      state: pda-cburn
+  - type: Icon
+    state: pda-cburn
```

Comment:
Is this needed?

Reply:
Probably not.
Honestly, I not sure what exactly this line does, as I'm not skilled enough, and cburn pda is a copypaste of another pda.

Reply:
you're basically just assigning a sprite to the pda. That sprite is the `pda-cburn` state you've created. Considering this item is so niche, it could probably reuse literally any of the existing pda icons.

Reply:
Ah, you are talking about the sprite itself. I misnterpret the question.
I decided to add it, as it took literally less than a minute to mix base sec pda and virology pda accent at the bottom.
You definitely right, i can do this same way ERT pda works.

But now you gave me an idea.
ERT pda uses HoS pda sprite. I dont like that idea much, because HoS is important, unique role, with the unique pda color scheme. And SecOff pda is too casual for a team of elites.
How about introducing new pda sprite, that will be used by both  CBURN and ERT (and any other, added in future, forces)?

Sprite attached.
Base color is red (security, protection), secondary color is blue (color of command, CentCom) cross (symbol of aid).
![pda-cburnalt](https://user-images.githubusercontent.com/100279397/187088810-6f38c2f7-3a08-4dac-a061-226d64768973.png)

Reply:
i think this would be preferable. the best solution would just be dynamically colored PDA sprites which. i think julian was doing but eh

Reply:
Made necessary changes